src/frontend/e2e/utils/schemas/sampleResponseData.ts

```diff
@@ -0,0 +1,38 @@
+export type SampleResponseData = {
+  id: number | undefined;
+  collection_date: string | undefined;
+  collection_location: {
+    id?: number;
+    region: string;
+    country: string;
+    division: string;
+    location: string;
+  };
+  czb_failed_genome_recovery?: boolean;
+  gisaid: {
+    gisaid_id: any;
```

Comment:
I don't believe this should be `any`

Reply:
The problem with this field is that currently in the backend this comes as "null" for every sample as a result, I don't know what type to use.
Also pls see my query: https://app.shortcut.com/genepi/story/216575/potential-typo-in-sample-response-from-backend

Reply:
OK, I see from GisaidResponseType that this is string, so updating